Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -490,6 +485,21 @@ void JitArm64::WriteExceptionExit(ARM64Reg dest, bool only_external)
   B(dispatcher);
 }
 
+bool JitArm64::HandleFunctionHooking(u32 address)
+{
+  return HLE::ReplaceFunctionIfPossible(address, [&](u32 hook_index, HLE::HookType type) {
+    HLEFunction(hook_index);
+
+    if (type != HLE::HookType::Replace)
+      return false;
+
+    LDR(INDEX_UNSIGNED, DISPATCHER_PC, PPC_REG, PPCSTATE_OFF(npc));
+    js.downcountAmount += js.st.numCycles;
```

Comment:
Honestly, I do not get the point why we add the cycles of this single instruction. I assume te HLE functions shall decrease downCount on their own, so this single instruction likely has no effect. However it is the same in the x64 JIT, so I'm fine with it.

Reply:
`js.st.numCycles` is for the whole block, not the current instruction. Though, implementing it that way also seems weird, considering that we might hit the HLE case halfway through a block after we've already added some cycles to `js.downcountAmount` the normal way...